src/elements/content-explorer/ContentExplorer.js

```diff
@@ -377,6 +387,38 @@ class ContentExplorer extends Component<Props, State> {
         }
     };
 
+    /**
+     * Call Box Representations API to fetch thumbnails of all files in collection.
+     * The thumbnailUrls array in state will be populated with urls to thumbnails.
+     * If a thumbnail could not be found, that item's entry in thumbnailUrls will be null.
+     *
+     * @private
+     * @param {Object} collection - item collection object
+     * @param {Boolean|void} triggerNavigationEvent - To trigger navigate event and focus grid
+     * @return {void}
+     */
+    async fetchThumbnailUrls(collection: Collection, dimensions: string): Promise<void> {
```

Comment:
If items is the only thing being used from `collection`, we can either deconstruct that in the function declaration or we can change the input to be the actual items array